core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -109,22 +109,30 @@ public String toString() {
   public static final PropertyKey CONF_DIR =
       new Builder(Name.CONF_DIR)
           .setDefaultValue(String.format("${%s}/conf", Name.HOME))
+          .setDescription("The directory containing files used to configure Alluxio.")
           .build();
   public static final PropertyKey DEBUG =
       new Builder(Name.DEBUG)
           .setDefaultValue(false)
+          .setDescription("Set to true to enable debug mode which has additional logging and "
+              + "info in the Web UI.")
           .build();
   public static final PropertyKey HOME =
       new Builder(Name.HOME)
           .setDefaultValue("/opt/alluxio")
+          .setDescription("Alluxio installation directory.")
           .build();
   public static final PropertyKey KEY_VALUE_ENABLED =
       new Builder(Name.KEY_VALUE_ENABLED)
           .setDefaultValue(false)
+          .setDescription("Whether the key-value service is enabled.")
           .build();
   public static final PropertyKey KEY_VALUE_PARTITION_SIZE_BYTES_MAX =
       new Builder(Name.KEY_VALUE_PARTITION_SIZE_BYTES_MAX)
           .setDefaultValue("512MB")
+          .setDescription("Maximum allowable size (in bytes) of a single key-value partition "
+              + "in a store. This value should be no larger than the block size "
+              + "(alluxio.user.block.size.bytes.default)")
```

Comment:
instead of hardcoding `alluxio.user.block.size.bytes.default`, use `Name.USER_BLOCK_SIZE_BYTES_DEFAULT`